justfile

```diff
@@ -20,9 +20,13 @@ alias bootstrap := new-project
 
 [group('bootstrapper')]
 new-project:
-  python3 -m pip install --user cookiecutter
-  python3 -m pip install --user Jinja2 jinja2-time
-  python3 -m cookiecutter git@github.com:thinknimble/tn-spa-bootstrapper.git
+  #!/usr/bin/env bash
+  if ! command -v pipx &> /dev/null; then
+    echo "ERROR: You must install pipx first: https://pipx.pypa.io/stable/installation/#installing-pipx"
+    exit 1
+  fi
+  pipx install cookiecutter
+  pipx run cookiecutter gh:thinknimble/tn-spa-bootstrapper
```

Comment:
@oudeismetis Here's the `pipx` version of the command.

In future, it could be made more convenient by auto-installing `pipx`, but that requires detecting the platform and so on, which is a lot of maintenance to sign up for this early on.

Aside, I'm thinking that as we ramp this up, we will limit support to only M-chip Macs and Ubuntu. We can easily test setup on Ubuntu from scratch using docker, but Macs will be trickier.

Reply:
you don't need to do `pipx run`. Once pipx installs something it should be available globally as it's own command (without need for terminal restart I believe)

Reply:
I agree for new projects like this we should focus on future support. It always annoyed me when I'd be starting a new project at a big company and then requirements would state that we HAD to support the last 2 versions of...say... IE (versions pinned by name), even though it was going to be a multi-year project and no one would be using those versions by the time we launched